### PR TITLE
[Bug fix]Bug fix for changing assertTrue to assertEqual

### PR DESCRIPTION
**Pull request by pruthvihebbani**
## Title
Bug fix to send valid PTI in ESM information response message

## Summary
ESM information response was sent with wrong Procedure Transaction Identity(PTI) due to which MME was sending STATUS message with cause "invalid PTI". This PR fixes the bug

## Test plan
1.Executed S1 SIM sanity
2.Executed test_attach_esm_information.py and test_attach_esm_information_wrong_apn.py TCs

### CODE REVIEW

File: TestCntlrApp/src/tfwApp/fw_api_int.c [hunk 1]
```diff
@@ -3131,6 +3131,7 @@ PRIVATE Void handleEsmInformationRsp(ueEsmInformationRsp_t* data)
    uetMsg->msgType = UE_ESM_INFORMATION_RSP_TYPE;
    ueEsmInfoRsp = &uetMsg->msg.ueEsmInformationRsp;
    ueEsmInfoRsp->ueId    = data->ue_Id;
+   ueEsmInfoRsp->tId     = data->tId;
```
Comment (ulaskozat) [inline TestCntlrApp/src/tfwApp/fw_api_int.c]: Nit: too many spaces before "=". Single space before and after.

File: TestCntlrApp/src/tfwApp/fw_api_int.c [hunk 2]
```diff
@@ -3131,7 +3131,7 @@ PRIVATE Void handleEsmInformationRsp(ueEsmInformationRsp_t* data)
    uetMsg->msgType = UE_ESM_INFORMATION_RSP_TYPE;
    ueEsmInfoRsp = &uetMsg->msg.ueEsmInformationRsp;
    ueEsmInfoRsp->ueId    = data->ue_Id;
```
Comment (ulaskozat) [inline TestCntlrApp/src/tfwApp/fw_api_int.c]: Can you also fix this one?

**Comment by ulaskozat** [comment]
I approved the PR, but make sure that line 3133 is also fixed.

File: TestCntlrApp/src/tfwApp/fw_api_int.c (hunk 2 above)
Comment (pruthvihebbani) [reply TestCntlrApp/src/tfwApp/fw_api_int.c]: done

File: TestCntlrApp/src/tfwApp/fw_api_int.c (hunk 1 above)
Comment (pruthvihebbani) [reply TestCntlrApp/src/tfwApp/fw_api_int.c]: done

**Pull request merged**